pkg/scheduler/plugins/deviceshare/utils.go

```diff
@@ -386,6 +386,12 @@ func parsePodDeviceShareExtensions(pod *corev1.Pod, podRequests map[schedulingv1
 
 	state.hints = hints
 	state.hintSelectors = hintSelectors
+	for _, selectors := range hintSelectors {
+		if selectors[0] != nil {
```

Comment:
can we assert`len(selector) > 0`?

Reply:
yes, 
![image](https://github.com/user-attachments/assets/df32a486-468e-46ab-bebb-c1bd5c4b25f1)